packages/core/src/rum/instrumentation/resourceTracking/distributedTracing/distributedTracingHeaders.ts

```diff
@@ -60,16 +61,26 @@ export const getTracingHeaders = (
                 break;
             }
             case PropagatorType.TRACECONTEXT: {
-                headers.push({
-                    header: TRACECONTEXT_HEADER_KEY,
-                    value: generateTraceContextHeader({
-                        version: '00',
-                        traceId: tracingAttributes.traceId,
-                        parentId: tracingAttributes.spanId,
-                        isSampled:
-                            tracingAttributes.samplingPriorityHeader === '1'
-                    })
-                });
+                headers.push(
+                    {
+                        header: TRACECONTEXT_HEADER_KEY,
+                        value: generateTraceContextHeader({
+                            version: '00',
+                            traceId: tracingAttributes.traceId,
+                            parentId: tracingAttributes.spanId,
+                            isSampled:
+                                tracingAttributes.samplingPriorityHeader === '1'
+                        })
+                    },
+                    {
+                        header: TRACESTATE_HEADER_KEY,
+                        value: generateTraceStateHeader({
+                            parentId: tracingAttributes.spanId,
+                            isSampled:
+                                tracingAttributes.samplingPriorityHeader === '1'
+                        })
+                    }
```

Comment:
Like I have been asking everyone, can you make sure that data in the datadog headers is same as tracecontext headers?

A test on it would be much appreciated if these isn't one.